File: 프로그래머스/1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.py
def GCD(n,m) :
    gcd=0
    for i in range(1,n+1,1) :
        if n%i ==0 and m%i ==0 :
            gcd = i
    return gcd

def LCM(n,m) :
    return (n*m) / GCD(n,m)

def solution(n, m):
    return [GCD(n,m),LCM(n,m)]

File: 프로그래머스/1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.java
class Solution {
    public int[] solution(int n, int m) {
        int[] answer = new int[2];
        
        int max_yak = 1;
        for(int i = 1; i<n+1; i++){
            if(n%i == 0 && m%i ==0){
                max_yak = i;
            }
        }
        answer[0] =max_yak;
        
        answer[1] = n*m/max_yak;
        
        
        return answer;
    }
}